### ISSUE TITLE
Style/RedundantFormat offense with width specifiers

### ISSUE THREAD

**Issue by shanecav84** [rubocop/rubocop#13866]
Style/RedundantFormat considers a format string with width specifiers to be an offense.
--------

## Expected behavior

Don't mark as an offense

## Actual behavior

Is marked as an offense

## Steps to reproduce the problem

```ruby
format('%10s', 'beepboop')
```

causes "Style/RedundantFormat: Redundant format can be removed"

## RuboCop version

```
1.72.0 (using Parser 3.3.7.1, rubocop-ast 1.38.0, analyzing as Ruby 3.4, running on ruby 3.4.1) [arm64-darwin24]
  - rubocop-capybara 2.21.0
  - rubocop-factory_bot 2.26.1
  - rubocop-rails 2.29.1
  - rubocop-rspec 3.4.0
  - rubocop-rspec_rails 2.30.0
```

**Comment by dvandersluis** [comment]
This was done on purpose because if you have a literal string it could just be padded itself:
`'  beepboop'`.

**Comment by koic** [comment]
Yep, considering the role and intent of this cop, I agree with @dvandersluis's opinion. On the other hand, adjusting the message might help reduce confusion in cases where `format` modifies the string. To address this, I have opened #13868 to adjust the message.

**Comment by shanecav84** [comment]
Here's a more realistic example:

```ruby
format(
  '%20s %5s %20s %10s %20s %10s %12s',
  'header1',
  'header2',
  'header3',
  'header4',
  'header5',
  'header6',
  'header7'
)
```

Is that redundant, and so should it be written like this?:

```ruby
"             header1 header2              header3    header4              header5    header6      header7"
```
